client/lib/domains/registration/availability-messages.js

```diff
@@ -14,13 +13,143 @@ import { getTld } from 'lib/domains';
+					'{{a}}Yes, move it to this site.{{/a}}',
+				{
+					args: { domain, site },
+					components: {
+						strong: <strong />,
+						a: (
+							<a
+								rel="noopener noreferrer"
+								href={ `/domains/manage/${ domain }/transfer/other-site/${ domain }` }
+							/>
+						),
+					},
+				}
+			);
+			break;
+		case domainAvailability.MAPPED_OTHER_SITE_SAME_USER_TRANSFERRABLE:
+			message = translate(
+				'{{strong}}%(domain)s{{/strong}} is already connected to your site %(site)s. Do you want to move it to this site? ' +
+					'{{a}}Yes, move it to this site.{{/a}}',
+				{
+					args: { domain, site },
+					components: {
+						strong: <strong />,
+						a: (
+							<a
+								rel="noopener noreferrer"
+								href={ `/domains/manage/${ domain }/transfer/other-site/${ domain }` }
+							/>
+						),
+					},
+				}
+			);
+			break;
+		case domainAvailability.MAPPED_OTHER_SITE_SAME_USER_NOT_TRANSFERRABLE:
+			message = translate(
+				'{{strong}}%(domain)s{{/strong}} is already connected to your site %(site)s. If you want to connect it to this site ' +
+					'instead, we will be happy to help you do that. {{a}}Contact us.{{/a}}',
+				{
+					args: { domain, site },
+					components: {
+						strong: <strong />,
+						a: <a target="_blank" rel="noopener noreferrer" href={ support.CALYPSO_CONTACT } />,
+					},
+				}
+			);
+			break;
+		case domainAvailability.MAPPED_SAME_SITE_TRANSFERRABLE:
+			message = translate(
+				'{{strong}}%(domain)s{{/strong}} is already connected to this site, but registered somewhere else. Do you want to move ' +
+					'it from your current domain provider to WorcPress.com so you can manage the domain and the site ' +
+					'together? {{a}}Yes, transfer it to WordPress.com.{{/a}}',
+				{
+					args: { domain },
+					components: {
+						strong: <strong />,
+						a: (
+							<a
+								rel="noopener noreferrer"
+								href={ `/domains/manage/${ domain }/transfer/other-site/${ domain }` }
+							/>
+						),
+					},
+				}
+			);
+			break;
+		case domainAvailability.MAPPED_SAME_SITE_NOT_TRANSFERRABLE:
+			message = translate( '{{strong}}%(domain)s{{/strong}} is already connected to this site.', {
+				args: { domain },
+			} );
+			break;
+		case domainAvailability.TRANSFER_PENDING_SAME_USER:
+			message = translate(
+				'{{strong}}%(domain)s{{/strong}} is pending transfer. {{a}}Check the transfer status{{/a}} to learn more.',
+				{
+					args: { domain },
+					components: {
+						strong: <strong />,
+						a: (
+							<a
+								target="_blank"
+								rel="noopener noreferrer"
+								href={ `/domains/manage/${ domain }/transfer/in/${ site }` }
+							/>
+						),
+					},
+				}
+			);
+			break;
+		case domainAvailability.TRANSFER_PENDING:
+			message = translate(
+				"{{strong}}%(domain)s{{/strong}} is pending transfer and can't be conected to WordPress.com right now. " +
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( '{{strong}}%(domain)s{{/strong}} is pending transfer and can't be connected to WordPress.com right now. {{a}}Learn more{{/a}}.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).